playbooks/ansible/roles/sit.cephfs/tasks/new_volume.yml

```
@@ -5,6 +5,16 @@
       ceph fs subvolume create sit_fs {{ name }}
         --mode {{ mode | default('0755') }}
   run_once: true
+  when: volume.cephfs.options.casesensitive is defined
+
+- name: Create the CephFS subvolume without casesensitivity
+  command: >
+    /root/cephadm shell --
+      ceph fs subvolume create sit_fs {{ name }}
+        --mode {{ mode | default('0755') }}
+          --casesensitive 0
+  run_once: true
+  when: volume.cephfs.options.casesensitive is not defined
```

Checking only if it's defined ignores its value. If we want to explicitly set `casesensitive: "no"`, it won't work. I think it would be better to define `casesensitive` as a boolean in _cluster-ceph.yml_ and use something like this:
```suggestion
  when: volume.cephfs.options.casesensitive | default(false)
```

---

Done.